backend/src/middlewares/check-user-owner/check-user-comment-owner.middleware.ts

```diff
@@ -0,0 +1,24 @@
+import { RequestHandler } from 'express';
```

Comment:
each middleware should be in the separate folder
`backend/src/middlewares/check-user-comment-owner/check-user-comment-owner.middleware.ts`